components/molecules/HighlightInput/highlight-input-form.tsx

```diff
@@ -0,0 +1,92 @@
+import Button from "components/atoms/Button/button";
+import { ChangeEvent, useEffect, useRef, useState } from "react";
+
+const HighlightInputForm = (): JSX.Element => {
+  const [isDivFocused, setIsDivFocused] = useState<boolean>(false);
+  const textAreaRef = useRef<HTMLTextAreaElement>(null);
+  const [bodyText, setBodyText] = useState<string>("");
+  const [row, setRow] = useState<number>(1);
+  const [title, setTitle] = useState<string>("");
+  const ref = useRef<HTMLFormElement>(null);
+  let rowLomit = 5;
+  let messageLastScrollHeight = textAreaRef.current ? textAreaRef.current?.scrollHeight : 50;
+
+  useEffect(() => {
+    const checkIfClickedOutside = (e: any) => {
```

Comment:
Provide a type here instead of `any`. What's being received by the callback?

```suggestion
    const checkIfClickedOutside = (e: any) => {
```